apps/mull-ui/src/app/app.tsx

```diff
@@ -41,6 +41,15 @@ export const App = () => {
             <EventPage event={dummyEvent} prevPage={'Review'} />
           </div>
         </Route>
+        {/* Temporary, to be removed */}
+        <Route exact path={'/test-event-card'}>
+          <div className="page-container">
+            <div style={{ padding: '1rem' }}>
```

Comment:
This padding should be part of `page-container` so all our pages have uniform padding and less repetition in our code.

Reply:
This might work for something like the create event page, but it wouldn't work for the event page, for example. I do agree that styling like this should be repeated as little as possible but idk how

Reply:
We could do what bootstrap does. They have a generic `container` class with default padding. Then you can use `container fluid` if you'd like the container to take up the full screen.

Reply:
Sure, I can do that real quick